### ISSUE TITLE
Can you offer me the trainning log?

### ISSUE THREAD

**Issue by songzijiang** [Deeachain/Segmentation-Pytorch#6]
I am trying to reproduce the scores, could you offer the log if it`s possible. So that I can check if the program is running correctly. Thank you.

**Comment by Deeachain** [comment]
1. When you need to get high metrics (or raw image size inference) in validation sets and test sets, annotation line 116 in script `/dataset/cityscapes/cityscapes.py` , which will default to using raw image size inference for input images. (The reason we use line 116 `tr.FixScaleCrop(crop_size=self.crop_size)` is that we validation every 20 epochs training, which doesn't take up A lot of computational resources. But the accuracy will be much lower.)
2. The next thing I care about is whether the inference results on the val set can be reproduced according to the weight of the model I trained.
3. In the process of reasoning, you can focus on the adjustment of the following bold superparameters, which can be helpful to the reasoning results.

> #!/bin/bash
python predict.py --model PSPNet_res50 \
                  --checkpoint /data/dingcheng/segment/checkpoint/cityscapes/PSPNet_res50/best_model.pth \
                  --out_stride 8 \
                  --root '/data/open_data' \
                  --dataset cityscapes \
                  --predict_type validation \
                  --predict_mode whole \
                  --crop_size 768 \
                  --**tile_hw_size** '768,768' \
                  --batch_size 2 \
                  --gpus 3 \
                  --**overlap** 0.3 \
                  --**scales** 0.5 0.75 1.0 1.25
4. PSPNet training log and super parameters, training equipment two pieces of 2080Ti. Finally, we inferred on the val set using the raw image size, and Miou got about 76.54%, which is not the highest index, and there is room for improvement (limited time and equipment, no more experiments were done).
```
+--------------+-------------------------------------------------------------------+
|  args_name   |                             args_value                            |
+--------------+-------------------------------------------------------------------+
|    model     |                            PSPNet_res50                           |
|   backbone   |                                None                               |
|  pretrained  |                               False                               |
|  out_stride  |                                 8                                 |
|  mult_grid   |                               False                               |
|     root     |                          /data/open_data                          |
|   dataset    |                             cityscapes                            |
| random_scale |                                True                               |
|  base_size   |                                768                                |
|  crop_size   |                                768                                |
| num_workers  |                                 16                                |
|  max_epochs  |                                200                                |
|  batch_size  |                                 4                                 |
|  val_epochs  |                                 20                                |
|      lr      |                                0.01                               |
|    optim     |                                sgd                                |
| predict_type |                             validation                            |
| predict_mode |                               whole                               |
| tile_hw_size |                              768,768                              |
|  flip_merge  |                               False                               |
|     loss     |                       ProbOhemCrossEntropy2d                      |
|     cuda     |                                True                               |
|  local_rank  |                                 0                                 |
|   gpus_id    |                                1,2                                |
|    resume    |                                None                               |
|   savedir    |               ./checkpoint/cityscapes/PSPNet_res50/               |
|   logFile    |                              log.txt                              |
|   classes    |                                 19                                |
| save_seg_dir |             ./checkpoint/cityscapes/PSPNet_res50/whole            |
|     seed     |                                 88                                |
|  parameters  |                            48.956262 M                            |
|     mean     |                  [72.3924   82.90902  73.158325]                  |
|     std      |                  [45.319206 46.15292  44.91484 ]                  |
| classWeights | [ 1.4705521  9.505282  10.492059  10.492059  10.492059  10.492059 |
|              |  10.492059  10.492059  10.492059  10.492059  10.492059  10.492059 |
|              |  10.492059  10.492059  10.492059  10.492059  10.492059  10.492059 |
|              |                              5.131664 ]                           |
+--------------+-------------------------------------------------------------------+
Best Validation Epoch 1 Best_Miou is 0.3971 OA is 0.8371
Best Validation Epoch 20 Best_Miou is 0.5587 OA is 0.9297
Best Validation Epoch 40 Best_Miou is 0.6061 OA is 0.9404
Best Validation Epoch 100 Best_Miou is 0.6532 OA is 0.9457
Best Validation Epoch 120 Best_Miou is 0.6568 OA is 0.9495
Best Validation Epoch 160 Best_Miou is 0.6815 OA is 0.955
Best Validation Epoch 180 Best_Miou is 0.6888 OA is 0.9568
Best Validation Epoch 191 Best_Miou is 0.7185 OA is 0.9598
Best Validation Epoch 194 Best_Miou is 0.7249 OA is 0.9614
Best Validation Epoch 198 Best_Miou is 0.7277 OA is 0.9619
Best Validation Epoch 200 Best_Miou is 0.7287 OA is 0.9615
```


If there is any progress in the experiment, I hope you can communicate with me and help me improve this project (this project needs a lot of improvement, which I have not paid attention to).

**Comment by songzijiang** [comment]
Thank you very much! The program is still running now. I will contract you when it finishes or I have any problems.
